components/LanguageSelector.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="relative px-1 py-2 cursor-pointer">
+    <div class="flex px-2 py-2 bg-gray-200" @click="showOptions" v-on-clickaway="closeOptions">
```

Comment:
add `rounded` for similar button "theme"
```suggestion
    <div class="flex px-2 py-2 bg-gray-200 rounded" @click="showOptions" v-on-clickaway="closeOptions">
```

Reply:
@samueljklee  Added. thanks!